airflow/models/dag.py

```diff
@@ -1816,18 +1839,9 @@ def get_paused_dag_ids(dag_ids, session):
     def safe_dag_id(self):
         return self.dag_id.replace('.', '__dot__')
 
-    def get_dag(self, store_serialized_dags=False):
-        """Creates a dagbag to load and return a DAG.
-        Calling it from UI should set store_serialized_dags = STORE_SERIALIZED_DAGS.
-        There may be a delay for scheduler to write serialized DAG into database,
-        loads from file in this case.
-        FIXME: remove it when webserver does not access to DAG folder in future.
-        """
-        dag = DagBag(
-            dag_folder=self.fileloc, store_serialized_dags=store_serialized_dags).get_dag(self.dag_id)
-        if store_serialized_dags and dag is None:
-            dag = self.get_dag()
-        return dag
+
+    def get_dag(self):
```

Comment:
looks like get_dag logic has changed.
This was in 1.10.4 (https://github.com/apache/airflow/blob/1.10.4/airflow/models/dag.py#L1508)

Here, you just need to replace `fileloc` with `get_local_fileloc()`, rest will be same